cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -124,12 +124,16 @@ func TestValidateIPFromString(t *testing.T) {
 		ip       string
 		expected bool
 	}{
-		{"", false},           // not valid
-		{"1234", false},       // not valid
-		{"1.2", false},        // not valid
-		{"1.2.3.4/16", false}, // not valid
-		{"1.2.3.4", true},     // valid
-		{"16.0.1.1", true},    // valid
+		{"", false},               // not valid
+		{"1234", false},           // not valid
+		{"1.2", false},            // not valid
+		{"1.2.3.4/16", false},     // not valid
+		{"1.2.3.4", true},         // valid
+		{"16.0.1.1", true},        // valid
```

Comment:
What condition is this testing, that is not tested by L131? Seems like a duplicate test in the original code.